reader.go

```diff
@@ -50,11 +50,16 @@ func (r *Reader) start() {
 }
 
 func (r *Reader) read() error {
-	resp, err := r.client.GetLogEvents(&cloudwatchlogs.GetLogEventsInput{
+
+	params := &cloudwatchlogs.GetLogEventsInput{
 		LogGroupName:  r.group,
 		LogStreamName: r.stream,
-		NextToken:     r.nextToken,
-	})
+		StartFromHead: aws.Bool(true),
+		NextToken: t.nextToken,
```

Comment:
This should be `r.`. Also [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports)!!!